test/utils/layout-helpers-test.ts

```diff
@@ -37,11 +37,13 @@ describe('definition-helpers', function () {
     it('return package root folder', async function () {
       const resultOne = await resolvePackageRoot(__dirname, 'lodash');
 
-      expect(resultOne).toContain(path.sep + 'lodash');
+      // no lodash here.
+      expect(resultOne).toBe(false);
 
       const resultTwo = await resolvePackageRoot(__dirname, 'memoizee');
 
-      expect(resultTwo).toContain(path.sep + 'memoizee');
```

Comment:
it's incorrect, we do have `memoizee` as straight language-server dependency, and helper should be able to find it.

<img width="725" alt="image" src="https://github.com/user-attachments/assets/f0f11037-baba-47aa-a6b1-25d68b2257b7">

Reply:
reverted